### PR TITLE
Implement 'traffic' endpoint

### PR DESCRIPTION
**Pull request by enderlein**

### CODE REVIEW

**Comment by alterNERDtive** [comment]
Hey! Thanks for the PR and I’m not generally opposed to merging, but please write some basic tests :)

I am aware that you cannot really test for specific data, but please have some that ensure you receive the format you expect.

**Comment by alterNERDtive** [comment]
Thanks, I will have a look tomorrow!

File: edsm/systemApi.py [hunk 1]
```diff
@@ -41,3 +41,42 @@ def getBodiesById(cls, systemId):
...
+  @classmethod
+  def getTrafficById(cls, systemId):
+    """
+    Requests information about traffic in a system.
+
+    :param systemName: ID of the system in question
+    """
+
+    json = cls.query({'systemId' : str(systemId)})
+    return json
```
Comment (alterNERDtive) [inline edsm/systemApi.py:82]: This might sound nitpicky, but this is missing a newline at the end :)

**Comment by alterNERDtive** [comment]
Sorry, forgot about this. Looks good apart from ↑ which should only be a minute of your time.

**Comment by alterNERDtive** [comment]
(I should add an editorconfig file.)

**Comment by enderlein** [comment]
You do have an editorconfig file, I just never installed the plugin. Oops.

**Comment by alterNERDtive** [comment]
😬

**Comment by alterNERDtive** [comment]
Want me to make a release for this? It’s not like I’m working on much else in here.

**Comment by enderlein** [comment]
Sure!

**Comment by alterNERDtive** [comment]
Sorry, took me a while because … well, I forgot about it.

https://github.com/alterNERDtive/pyEDSM/releases/tag/releases%2F0.2

**Pull request merged**